department-of-veterans-affairs/va.gov-cms
# VACMS-3605: CLP Event Panel

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/3605

## Testing done


## Screenshots

![Screen Shot 2020-12-04 at 2 44 37 PM](https://user-images.githubusercontent.com/56809719/101207907-57465a00-363f-11eb-89e6-db3c1258a748.png)

![Screen Shot 2020-12-04 at 2 45 06 PM](https://user-images.githubusercontent.com/56809719/101207911-59a8b400-363f-11eb-91d6-33d94b47e3af.png)

## QA steps

As user _uid_ with _user_role_

- Login into PR environment
- Navigate here: /node/add/campaign_landing_page
- Select the 'Events' panel tab
- [ ] Confirm no fields appear other than **Add an Events panel** boolean
-  Select **Add an Events panel** boolean and confirm **Events Header*** and **ADD UP TO TWO EVENTS** appear.
-  Enter a value in the **Events Header**
-  Select the **ADD UP TO TWO EVENTS** fieldset and select **Select Events** button
- [ ] Confirm **Select Events** entity browser appears with 2 filters: Owner and Title.
-  Select 2 events and select **Select Entities** .
- Complete remainder of tabs on form
- Confirm **Save** is successful

### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
